src/Components/HistoricalDataChart/HistoricalDataChart.js

```diff
@@ -119,7 +121,22 @@ export const HistoricalDataChart = ({ inventoryId }) => {
                         containerComponent={
                             <VictoryZoomVoronoiContainer
                                 labels={({ datum }) => {
-                                    return datum.childName.includes('scatter-') && datum.y !== null ? `${datum.name}: ${datum.y}%` : null;}
+                                    const isToday = new Date().toDateString() === new Date(datum.x).toDateString();
+                                    let xAxisDate = isToday ? 'Today' : `${new Date(datum.x).getDate()} ${MONTHS[new Date(datum.x).getMonth()]}`;
```

Comment:
@PreetiW,
Here, you have created date object three times by passing the same value i.e. `datum.x`.
Though `new Date(datum.x)` objects are garbage collected automatically, will it be good to capture value in a variable & reuse it wherever required?

Reply:
good catch @kgaikwad I have updated the PR by creating a single variable and pushed the changes.